src/App.vue

```diff
@@ -158,21 +157,16 @@ import { useStore } from '@/store/store'
     SbcFooter,
     ...Dialogs,
     ...Views
-  },
-  mixins: [
-    CommonMixin,
-    FilingTemplateMixin
-  ]
+  }
 })
-export default class App extends Vue {
+export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
   // Refs
   $refs!: {
     confirm: ConfirmDialogType
   }
 
   // Global getters
   @Getter(useStore) getAppValidate!: boolean
-  @Getter(useStore) getBusinessId!: string
```

Comment:
This getter reference is redundant because we get one from a mixin. (This caused build errors.)